gateway/http.d/apicast.conf.liquid

```diff
@@ -14,6 +14,13 @@ map "" $extended_access_logs_enabled {
   default '0';
 }
 
+map "" $post_action_impact {
+  default 0;
```

Comment:
I think the value is defined as a string `'0'`

Reply:
I've set it to empty string to reflect [this](https://github.com/3scale/APIcast/blob/e17e421944bdffe97366e306123a83c2874720dd/gateway/conf.d/apicast.conf#L159)